os/arch/arm/src/amebad/amebad_reboot_reason.c

```diff
@@ -88,6 +88,21 @@ static reboot_reason_code_t up_reboot_reason_get_hw_value(void)
 		else if ((boot_reason & BIT_BOOT_KM4WDG_RESET_HAPPEN) || (boot_reason & BIT_BOOT_WDG_RESET_HAPPEN)) {
 			return REBOOT_SYSTEM_WATCHDOG;
 		}
+
+		/* KM4 deep sleep handled by KM0 (KM4 sleep + KM0 tickless, KM4 deep sleep + KM0 deep sleep AON) */
+		else if (boot_reason & BIT_BOOT_DSLP_RESET_HAPPEN) {
+			return REBOOT_SYSTEM_DSLP_RESET;
+		}
+
+		/* KM4 or KM0 WDT reset */
```

Comment:
This comment from WDT should be sys reset?

Reply:
Sorry, is a copy paste mistake.
Corrected.